src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -148,15 +148,20 @@
     <RuntimeGroup Include="ol">
       <Parent>rhel</Parent>
       <Architectures>x64</Architectures>
-      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6</Versions>
+      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6;7.7;7.8;7.9;7.10</Versions>
```

Comment:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.9_release_notes/overview

>  Red Hat Enterprise Linux 7.9 is the last minor release of RHEL 7.

We shouldn't need to add 7.10 here.